### DISCUSSION TITLE
[BUG] [RBAC] default for header is not actually working

### DISCUSSION THREAD

**Discussion by giuliowaitforitdavide** [mia-platform/community#105] [Q&A]
![Schermata del 2022-11-17 13-07-47](https://user-images.githubusercontent.com/50294725/202677284-c6d06835-43d5-405a-911c-8c522f22c084.png)
After the release of 10.1.0 I'm not able to save RBAC rules if the header filter field is empty, even if a default should be setted (`acl_rows` as in the image). In other rules, saved before the 10.1.0 release, this field is empty and it works fine.

**Comment by davidebianchi** [comment]
Hi @giuliowaitforitdavide! It seems a bug. @albertotessarotto @fredmaggiowski  we should check and resolve this. To save at the moment, what if you enter `acl_rows` manually?

**Reply by giuliowaitforitdavide** [reply]
By manually setting the value it works. My bad I didn't write it

**Reply by fredmaggiowski** [reply]
It seems that the regex is not accepting the `_`, we've changed some validation and this might have slipped; we're fixing it ASAP; thank you for finding this out!

**Reply by albertotessarotto** [reply]
Thanks for the issue, we will fix it soon

**Comment by eros-piccioni** [comment]
Hi, I also have a related issue. I'm not able to update a manual route even if the "Apply filter on request" is not enabled. The header name for row filter validation prevent the update.

![image (1)](https://user-images.githubusercontent.com/40391627/202692116-e93895d6-6bee-4bc2-a384-196045ab29aa.png)

**Reply by albertotessarotto** [reply]
Thanks for having reported here, we will fix it soon

**Comment by fredmaggiowski** [comment]
This bug has been fixed and will be featured in the next Console release (approximately next thursday
